workbench-core/datasets/src/errors/invalidPermissionError.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+export class InvalidPermissionError extends Error {
```

Comment:
add to the index file

Reply:
so many many things I neglected to export! Fixed!